solutions/standard/variables.tf

```diff
@@ -225,6 +225,13 @@ variable "kms_key_name" {
   description = "The name for the new root key. Applies only if `existing_secrets_manager_kms_key_crn` is not specified. If a prefix input variable is passed, it is added to the value in the `<prefix>-value` format."
 }
 
+variable "ibmcloud_kms_api_key" {
+  type        = string
+  description = "The IBM Cloud API key with access to create a root key and key ring in the key management service instance. If the KMS instance is in a different account, specify a key from that account. If not specified, the ibmcloud_api_key variable is used."
```

Comment:
A bit long winded, but I was thinking something like this:

If the instance passed in `existing_kms_instance_crn` is located in a different account to the secrets manager instance, pass an apikey from the KMS account that has access to create a root key. If KMS is in the same account as the secrets manager instance, do no pass any value for this variable so it will default to the value passed in the `ibmcloud_api_key` variable.

@SirSpidey you might do a better job at wording this if you don't mind please?

Reply:
@ocofaigh Try this:

> The IBM Cloud API key that can create a root key and key ring in the key management service (KMS) instance. If not specified, the `ibmcloud_api_key` variable is used. Specify this key if the instance in `existing_kms_instance_crn` is in an account that's different from the Secrets Manager instance. Leave this input empty if the same account owns both instances.

also
**existing_kms_instance_crn**
description = "The CRN of the KMS instance (Hyper Protect Crypto Services or Key Protect). Applies only if `existing_secrets_manager_kms_key_crn` is not specified. If the KMS instance is in different account you must also provide a value for `ibmcloud_kms_api_key`"